caffe2/python/operator_test/asin_op_test.py

```diff
@@ -0,0 +1,31 @@
+from __future__ import absolute_import
```

Comment:
looks like asin and acos can go together and reuse the same test with except passing different lambda for reference as well as different operator name, what do you think?